### PR TITLE
App Catalog --> Apps and Integrations

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

Name change per @stacykor: App Catalog --> Apps and Integrations.

Issue number: 

## Select the type of change:

- [x] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/integrations/index.md [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
-tags: [app, app catalog]
+tags: [app, apps, app catalog, integrations]
 ---
 
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
 <img src={useBaseUrl('img/integrations/integrations.png')} alt="icon" width="50"/>
 
-This category has documentation for Sumo Logic apps, which you can find in our **App Catalog**. Apps help you get started gaining insights from your data sources by providing example searches and dashboards for common use cases. They are customized with your source configurations and populated in a folder selected by you. Feel free to edit them to get the results you want.
+This category has documentation for Sumo Logic apps and integrations, located in our **App Catalog**,
```
Comment (stacykor) [inline docs/integrations/index.md:12]:
```suggestion
This category has documentation for Sumo Logic apps, webhooks, and other integrations, located in our **App Catalog**,
```

**Review by jpipkin1 — CHANGES_REQUESTED** [review]
@kimsauce, instead of "Apps, Integrations" for the label, I'd like to see "Apps and Integrations". I know it's wider, but it's more readable.

**Review by stacykor — COMMENTED** [review]
Apps, webhooks, and source only integrations will go in here so my preference is to call out webhooks by name.

File: docusaurus.config.js [hunk 2]
```diff
@@ -297,7 +297,7 @@ module.exports = {
               {
                 type: 'docSidebar',
                 sidebarId: 'integrations',
-                label: 'App Catalog',
+                label: 'Apps, Integrations',
```
Comment (kimsauce) [inline docusaurus.config.js:300]:
```suggestion
                label: 'Apps and Integrations',
```

**Comment by kimsauce** [comment]
> instead of "Apps, Integrations" for the label, I'd like to see "Apps and Integrations". I know it's wider, but it's more readable.

@jpipkin1 done!

**Review by jpipkin1 — APPROVED** [review]

**Pull request merged**